pkg/network/ovn_kubernetes.go

```diff
@@ -52,6 +52,17 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, manifestDir string) ([]*uns.U
 		svcpools += fmt.Sprintf("%s", net)
 	}
 	data.Data["OVN_service_cidr"] = svcpools
+	data.Data["OVN_namespace"] = "openshift-ovn-kubernetes"
+
+	// ovnkube config file
+	data.Data["K8S_cluster_subnets"] = ippools
+	data.Data["OVNKUBE_loglevel"] = 2
```

Comment:
The config file hardcodes the loglevel, so we should be consistent. Either hardcode it in the config and don't bother templating, or template it all the way through and not hardcode it in the config file.

Reply:
@dcbw Its now templated.